index.html

```diff
@@ -0,0 +1,152 @@
+    />
+    <title>Medical Core</title>
+
+</head>
+
+<body class="body">
+    <header class="header">
+
+        <nav class="nav">
+            <div class="nav__logo">
+                <img src="img/Medical Core.png" alt="">
+
+            </div>
+            <ul class=" nav__list nav-list">
+                <li class="nav__item">
+                    <a href="/#" class="nav__link"> Home</a></li>
+                <li class="nav__item"><a href="/#" class="nav__link"> Product</a></li>
+                <li class="nav__item"><a href="/#" class="nav__link"> Pricing</a></li>
+                <li class="nav__item"><a href="/#" class="nav__link"> Contact</a></li>
+            </ul>
+        </nav>
+        <div class="buttons buttons--spacebtn">
+            <a href="#" class="button ">Login</a>
+            <a href="#" class="button button--colorbackground">Join Us <span class=" arrow">  &#8594;</span></a>
+        </div>
+
+    </header>
+    <section class="medpage medpage--welcomepage">
+        <p class="medpage__welcome">Welcome</p>
+        <h1 class="medpage__title">A Great Place <br> To Receive Care</h1>
+        <p class="medpage__subtitle">Get your best looking smile now!</p>
+
+
+        <div class="buttons ">
+            <a href="#" class=" button button--colorbackground ">Join Us</a>
+
+            <a href="#" class="button button--border">Learn More</a>
+        </div>
+
+
+    </section>
+    <section class="medpage medpage--experts">
+
+        <div class="medpage__img">
+            <img src="img/thumb-concept.png" alt="" class="medpage__picture">
+        </div>
+        <div class="medpage__header">
+
+            <hr class="lineblock">
+            <h1 class="medpage__title medpage__title--experts">Meet our Experts</h1>
+            <p class="medpage__subtitle medpage__subtitle--experts">Problems trying to resolve the conflict between <br>the two majorrealms of Classical physics: <br> Newtonian mechanics </p>
+            <a href="#" class="button button-section">Learn More <i class="fa-solid fa-chevron-right"></i></a>
+
+        </div>
+
+
+    </section>
+    <section class="medpage medpage--ourteam">
+        <div class="medpage__team">
+            <p class="medpage__welcome">Our Team</p>
+
+            <h1 class="medpage__title medpage__title--smallfont">Our Team</h1>
+            <p class="medpage__subtitle ">Problems trying to resolve the conflict between <br> the two major realms of Classical physics: Newtonian mechanics </p>
+            <!--  -->
+
+            <!-- <a href="#" class=" button button--bg-color ">Join Us</a> -->
+            <!-- button--bg-none -->
+            <a href="#" class="button button-section">Learn More </a>
+
+        </div>
+        <div class="medpage-cards">
```

Comment:
`medpage-cards` is some how not meaning full. May be use `medpage__cards`. `cards` being an element of `medpage`